infrastructure/tests/test_data_import.py

```diff
@@ -124,3 +187,111 @@ def test_year_error(self):
         self.assertEquals(AnnualSpendFile.objects.all().count(), 1)
         spend_file = AnnualSpendFile.objects.first()
         self.assertEquals(spend_file.status, AnnualSpendFile.ERROR)
+
+
+    def test_upload_project(self):
+        """Scope of Test: With no existing projects run an upload and check that the correct fields are populated"""
+        self.assertEquals(BudgetPhase.objects.all().count(), 5)
```

Comment:
what's this for?

Reply:
What I tried to do is check that the expected amount of BudgetPhases are created - to prevent cases like that bug where a budget column was not being created/updated for Full Year Forecast

Reply:
It's not obvious why this is relevant to the test - why the number of budget phases in the db before the test is relevant to the test. Perhaps a comment would help. I still don't totally understand.